routemaster/state_machine/tests/test_actions.py

```diff
@@ -62,6 +66,62 @@ def test_actions_do_not_advance_state_on_fail(app_config, create_label, mock_web
     ])
 
 
+def test_actions_retries_use_same_idempotency_token(app_config, create_label, mock_webhook, assert_history):
```

Comment:
Would be good to have a test which validates the inverse -- that different actions get different tokens.

Reply:
Added in fe9dc27608b651d92912681589bf12f6744a6668